src/map/utils/battleutils.cpp

```diff
@@ -1513,9 +1513,8 @@ namespace battleutils
     }
 
     //todo: need to penalise attacker's RangedAttack depending on distance from mob. (% decrease)
-    float GetRangedPDIF(CBattleEntity* PAttacker, CBattleEntity* PDefender)
+    float GetRangedDamageRatio(CBattleEntity* PAttacker, CBattleEntity* PDefender, bool isCritical)
```

Comment:
Function name changed to reflect the GetDamageRatio() function, which serves the same purpose.